src/components/ContactCard.tsx

```diff
@@ -0,0 +1,25 @@
+import '../styles/ContactCard.scss';
+
+const ContactCard = (props: any) => {
+    const favoriteIcon = "./images/favorite.svg";
+    const { nameSurname, position, job, mobile, favorite } = props.userContact;
```

Comment:
If userContact is the only field you need from props you could destructure it right on enter
`const ContactCard = ({ userContact }: Props) => {`
But to do this you'll need to define a Props interface first:
```
interface Props {
  userContact: { 
  nameSurname: string
...
  }
}
```

Reply:
yes, only userContact data used in 'render' method. Already added in a new request, thanks.